README.md

```diff
@@ -30,16 +30,12 @@ and [Cassandra](http://cassandra.apache.org/)/[ScyllaDB](https://www.scylladb.co
 
 ## Let's try it!
 
-**This is the master branch, which is not guaranteed to always be in a usable state.**
-
-**For production purposes we recommend using the latest stable release (currently [v1.1](https://github.com/astarte-platform/astarte/tree/release-1.1)), this branch should be used only for v1.2 development activities.**
-
```

Comment:
We will remove these lines once 1.2 is stable. For the moment, since we're in rc, let's keep them